prover/benches/row_matrix.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use math::{fft, fields::f64::BaseElement, StarkField};
+use rand_utils::rand_vector;
+use std::time::Duration;
+use winter_prover::{Matrix, RowMatrix, StarkDomain};
+
+// CONSTANTS
+// ================================================================================================
+
+const SIZE: usize = 524_288;
+const BLOWUP_FACTOR: usize = 8;
+const NUM_POLYS: [usize; 4] = [16, 32, 64, 96];
```

Comment:
We can probably get rid of the case for 16 polys (just keep 32, 64, and 96) - but I think it would be cool benchmark a different blowup factor too. Specifically, blowup factor of 2.